benchmarks/setup.py

```diff
@@ -125,11 +125,33 @@ def run(self):
         super().run()
 
 
-setup(name="triton-kernels-benchmark", packages=[
-    "triton_kernels_benchmark",
-], package_dir={
-    "triton_kernels_benchmark": "triton_kernels_benchmark",
-}, package_data={"triton_kernels_benchmark": ["xetla_kernel.cpython-*.so"]}, cmdclass={
-    "build_ext": build_ext,
-    "clean": clean,
-}, ext_modules=[CMakeExtension("triton_kernels_benchmark")])
+def get_git_commit_hash(length=8):
+    try:
+        cmd = ["git", "rev-parse", f"--short={length}", "HEAD"]
+        return "+git{}".format(subprocess.check_output(cmd).strip().decode("utf-8"))
+    except Exception:
+        return ""
+
+
+def get_install_requires():
+    install_requires = ["torch", "matplotlib", "pandas", "tabulate"]  # yapf: disable
+    return install_requires
+
+
+setup(
+    name="triton-kernels-benchmark",
+    version="3.1.0" + get_git_commit_hash(),
+    packages=["triton_kernels_benchmark"],
+    install_requires=get_install_requires(),
```

Comment:
Nit: the function is not required
```suggestion
    install_requires=["torch", "matplotlib", "pandas", "tabulate"],
```